gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -59,7 +58,6 @@
  *       had no outstanding demand. Duration.ZERO disables the timeout.
  * </ul>
  */
-@InternalApi
```

Comment:
EndUsers should really not be setting Watchdogs directly. However if you insist on continuing on with this PR, at least mark this as BetaApi, since its api was never reviewed for public consumption and please mark it as final so that users don't try to extend it

Reply:
But wouldn't marking a class as `@BetaApi` considered breaking change?

Reply:
My understanding is that since its marked InternalOnly, we are allowed to change this class in any way we see fit.